packages/main/src/plugin/kubernetes/configmaps-resource-factory.ts

```diff
@@ -54,6 +54,6 @@ export class ConfigmapsResourceFactory extends ResourceFactoryBase implements Re
     const apiClient = kubeconfig.getKubeConfig().makeApiClient(CoreV1Api);
     const listFn = (): Promise<V1ConfigMapList> => apiClient.listNamespacedConfigMap({ namespace });
     const path = `/api/v1/namespaces/${namespace}/configmaps`;
-    return new ResourceInformer<V1ConfigMap>(kubeconfig, path, listFn, 'configmaps');
+    return new ResourceInformer<V1ConfigMap>(kubeconfig, path, listFn, { kind: 'ConfigMap', plural: 'configmaps' });
```

Comment:
issue: I think we reached the Point of No Return in the parameter list

so it's easy to add/remove stuff and to identify the parameters and the order is not mandatory
new ResourceInformer<V1ConfigMap>({kubeConfig, path, listFn, kind, plural})

Reply:
I think you won 🎉 🎁